### PR TITLE
common: add SERIAL_CONTROL column to PixRacer serial table

### PR DESCRIPTION
**Pull request by peterbarker**
Also remove NuttX/PX4 information

### CODE REVIEW

**Comment by Hwurzburg** [comment]
there is no indication what the asterisk on SERIAL_CONTROL means nor what SERIAL_CONTROL really is ....a link to copter-commands-in-guided-mode.rst with directions to the command or https://mavlink.io/en/messages/common.html#SERIAL_CONTROL would make it clearer

I think USART8 line should be removed....its UART8 and is marked as Frsky telem on the board docs and is SERIAL4 which has GPS defaults in the code so protocol would have to be changed to use it in Frsky mode, perhaps a note?

USART1 is SERIAL5, UART7 is SERIAL6

**Comment by peterbarker** [comment]
> there is no indication what the asterisk on SERIAL_CONTROL means nor what SERIAL_CONTROL really is ....a link to copter-commands-in-guided-mode.rst with directions to the command or https://mavlink.io/en/messages/common.html#SERIAL_CONTROL would make it clearer

I've copied the note in from the PixHawk page.

> I think USART8 line should be removed....its UART8 and is marked as Frsky telem on the board docs and is SERIAL4 which has GPS defaults in the code so protocol would have to be changed to use it in Frsky mode, perhaps a note?

It's USART8 on the schematics.  SERIAL4 is correct.  Given the inverted nature of this port, I expect we should add 
```
define HAL_SERIAL4_PROTOCOL SerialProtocol_FrSky_Passhthrough to the hwdef
```

> USART1 is SERIAL5, UART7 is SERIAL6

I've added that data.

I pulled these PRs from 2-year-old branches as part of a cleanup.  I should have looked at this one a bit more closely...

I think there's some value in this page - right up until the point we do this properly and generate pages from the hwdefs.  I thought @tridge had done something along those lines already, but the extra notes about how they're marked on the case would not have been present.

**Comment by kd0aij** [comment]
actually SERIAL4 is selectively invertible on pixracer/fmuv4
SERIAL4_OPTIONS=2 turns off the inverter on both receive and transmit pins

File: common/source/docs/common-pixracer-serial-names.rst [hunk 1]
```diff
@@ -8,22 +8,27 @@ Many different names are given to the serial hardware on PixRacer
...
-+-----------+----------------+---------+-------------+------------+--------------+----------------+
-| UART8     | ?              | ?       | \SERIAL4_   | GPS        | GPS          |                |
-+-----------+----------------+---------+-------------+------------+--------------+----------------+
-| UART7     | ?              | ?       | ?           | DEBUG      | Debug        |                |
-+-----------+----------------+---------+-------------+------------+--------------+----------------+
-| USART1    | ?              | ?       | ?           | ESP        | ESP8266      | bl serial      |
-+-----------+----------------+---------+-------------+------------+--------------+----------------+
-| USART8    | ?              | ?       | ?           | FRS        | FRSky        | invertible     |
-+-----------+----------------+---------+-------------+------------+--------------+----------------+
++-----------+-------------+------------+--------------+-----------------+----------------+
```
Comment (tridge) [inline common/source/docs/common-pixracer-serial-names.rst:11]: drop the SERIAL_CONTROL column

File: common/source/docs/common-pixracer-serial-names.rst [hunk 2]
```diff
@@ -8,22 +8,27 @@ Many different names are given to the serial hardware on PixRacer
...
+| DataSheet | ParamPrefix | HW Label   | Def Role     | SERIAL_CONTROL* | Notes          |
```
Comment (rmackay9) [inline common/source/docs/common-pixracer-serial-names.rst:12]: an existing issue but we should probably call this column, "Default Protocol" so we stick with words that we use in other places.

File: common/source/docs/common-pixracer-serial-names.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-pixracer-serial-names.rst:11]: bl serial needs explanation

File: common/source/docs/common-pixracer-serial-names.rst [hunk 3]
```diff
@@ -8,22 +8,27 @@ Many different names are given to the serial hardware on PixRacer
...
++===========+=============+============+==============+=================+================+
+|           | \SERIAL0_   | USB        | Console      |                 | Micro-USB Plug |
++-----------+-------------+------------+--------------+-----------------+----------------+
+| UART4     | \SERIAL3_   | GPS        | GPS          | ?               |                |
++-----------+-------------+------------+--------------+-----------------+----------------+
+| USART2    | \SERIAL1_   | TELEM1     | MAVLink      | ?               |                |
++-----------+-------------+------------+--------------+-----------------+----------------+
+| USART3    | \SERIAL2_   | TELEM2     | MAVLink      | ?               |                |
++-----------+-------------+------------+--------------+-----------------+----------------+
+| UART7     | \SERIAL6    | DEBUG      | Debug        | 10              |                |
```
Comment (Hwurzburg) [inline common/source/docs/common-pixracer-serial-names.rst:22]: Param prefix notation inconsistent

**Comment by peterbarker** [comment]
Closing in favour of https://github.com/ArduPilot/ardupilot_wiki/pull/2833

**Comment by Hwurzburg** [comment]
replaced by #2833....fixes from reviewers incorporated...

**Pull request closed without merging**